src/SignalR/server/Core/src/Internal/DefaultHubDispatcher.cs

```diff
@@ -896,4 +897,4 @@ private static void SetActivityError(Activity? activity, Exception ex)
         activity?.SetTag("error.type", ex.GetType().FullName);
         activity?.SetStatus(ActivityStatusCode.Error);
     }
-}
+}
```

Comment:
Nit: restore the final newline.
```suggestion
}

```